server/src/main/resources/plugins/scripts/util/environment-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/sh
+
+#!/bin/sh
+
+function check () {
+  echo "Check if $1 is available."                                                      
+  if ! [ -x "$(command -v $1)" ]; then                                                 
```

Comment:
You might never know if given program has a space in it. Please use `"$1"` instead of `$1'